src/app/modules/showcase-forms/examples/form-field-select-example/form-field-select-example.component.ts

```diff
@@ -86,6 +86,65 @@ export class FormFieldSelectExampleComponent {
               ],
             },
           },
+          {
+            attribute: 'forthPropName',
+            title: 'May the force be with you',
+            editType: EditType.Select,
+            options: {
+              selectOptions: [
+                {
+                  value: 'may',
+                  label: 'May',
+                },
+                {
+                  value: 'the',
+                  label: 'the',
+                },
+                {
+                  value: '4th',
+                  label: '4th',
+                },
+              ],
+              clearFieldButton: {
+                enabled: true,
+                click: (fieldControl) => {
+                  fieldControl.setValue('none');
+                  fieldControl.markAsTouched();
+                  fieldControl.markAsDirty();
```

Comment:
Why are you using a custom function for this? Shouldn't this just happen in the library itself? And why do you want to set the value to a string value 'none' instead of null?

Reply:
No, this is just an example on how you can leverage the `clearFieldButton`, but the implementation will depend from project to project.
For the STIP project we need to send some value to the BE, by setting the desired clear value to send, without this value being on the selectOptions, the form will have the value but the field will be cleared, this gives the user feedback on what just happened and saves the correct value to send to the BE. Hope is clear

Reply:
I still think it would make sense to have a default click handler which simply clears the field and marks it as dirty and touched since this is what will be needed in most projects. You can then just override it if you want to do something else.

Reply:
ok, I'll try to do it 👌

Reply:
Done, Ive added a default action